hms-app/src/pages/MyIdeas.tsx

```diff
@@ -61,7 +61,8 @@ export default function MyIdeas() {
 
   const filteredIdeas = relevantIdeas.filter((item) => {
     const userId = user?.id || ''
-    return item.owner?.id.includes(userId)
+    const hackathon = hackathonData.id
+    return item.owner?.id.includes(userId) && item.hackathon?.id.includes(hackathon)
```

Comment:
why is the check that it `includes` the id and not that it equals (`===`) the id?

Reply:
Good point, will change it ASAP